src/data/data_loader.py

```diff
@@ -35,21 +35,24 @@ def __init__(self, dataset, use_batch=True, batch_size=100, n_batches=0, shuffle
         self.n_batches = settings.get('n_batches', n_batches)
         self.shuffle = settings.get('shuffle', shuffle)
         self.sampling_ratio = settings.get('sampling_ratio', sampling_ratio)
-        self.cell_number = self.cc.settings['general']['distribution']['client_id']
+        cell_settings = self.cc.settings['general']['distribution']
+        self.cell_number = cell_settings.get('client_id', 0)
 
-    def load(self):
+    def load(self, train=True):
         # Image processing
 
         # Dataset
         dataset = self.dataset(root=os.path.join(self.cc.settings['general']['output_dir'], 'data'),
-                               train=True,
+                               train=train,
                                transform=self.transform(),
                                download=True)
 
         if self.sampling_ratio >= 1:
             self.sampler = None
         else:
-            set_random_seed(self.cc.settings['general']['seed'] + self.cell_number,
+            # set_random_seed(self.cc.settings['general']['seed'] + self.cell_number,
+            #                 self.cc.settings['trainer']['params']['score']['cuda'])
```

Comment:
Oh, at the time, I think I was receiving errors about being unable to find client_id in the settings, but this has been fixed since in the latest version of master. Now that I have merged the latest version of master into my branch I will uncomment this and use the latest version!